hack/openapi.sh

```diff
@@ -19,4 +19,4 @@ source ./hack/go-path.sh
 # get the openapi binary
 command -v openapi-gen >/dev/null || go build -o "${GOPATH}"/bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
 echo "Generating openapi files"
-openapi-gen --logtostderr=true -v 1 -o "" -i github.com/kiegroup/kogito-cloud-operator/api/v1beta1 -O zz_generated.openapi -p ./api/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
+openapi-gen --logtostderr=true -v 1 -o "" -i github.com/kiegroup/kogito-cloud-operator/api/v1beta1 -O zz_generated.openapi -p ./api/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
```

Comment:
```suggestion
openapi-gen --logtostderr=true -v 1 -o "" -i github.com/kiegroup/kogito-cloud-operator/api/v1beta1 -O zz_generated.openapi -p ./api/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
openapi-gen --logtostderr=true -v 1 -o "" -i github.com/kiegroup/kogito-cloud-operator/api/v1 -O zz_generated.openapi -p ./api/v1 -h ./hack/boilerplate.go.txt -r "-"
```

Reply:
Currently there is no package for `v1`. May be we could introduce it once we start working on product operator